src/components/ContentWrapper.jsx

```diff
@@ -39,22 +40,27 @@ const ContentWrapper = (props) => {
   const route = router?.route || '';
 
   const {
-    loading: pipelineLoading,
-    error: pipelineError,
-    status: workerAndPipelineStatus,
+    loading: backendLoading,
+    error: backendError,
+    status: backendStatus,
```

Comment:
I am not seeing in this PR where actually `pipelineError`, `pipelineLoading`, ... etc got renamed to `backend...` -- is it part of a different PR?

Reply:
nevermind, it's fine, I just got confused